smartBugs.py

```diff
@@ -16,22 +16,22 @@
 from time import time, localtime, strftime
 
 
-cfg_dataset_path = os.path.abspath('config/dataset/dataset.yaml')
+cfg_dataset_path = os.path.abspath(os.path.dirname(os.path.realpath(__file__)) + '/config/dataset/dataset.yaml')
```

Comment:
consider creating a const variable at the top:

```FILE_PATH=os.path.dirname(os.path.realpath(__file__)```

or something along these lines.

Reply:
Also, you can then have another one

```
DATASET_CONFIG_FILE_PATH = FILE_PATH + '/config/dataset/dataset.yaml'
```

This would read better.